jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -30,16 +30,30 @@ YUI.add('user-profile-entity', function() {
         React.PropTypes.object,
         React.PropTypes.array
       ]),
+      d3: React.PropTypes.object,
       displayConfirmation: React.PropTypes.func,
       entity: React.PropTypes.object.isRequired,
       expanded: React.PropTypes.bool,
       getDiagramURL: React.PropTypes.func,
+      getKpiMetrics: React.PropTypes.func,
       permission: React.PropTypes.string,
       switchModel: React.PropTypes.func,
       type: React.PropTypes.string.isRequired
     },
 
     /**
+      Set initial state for KPI metrics
+    */
+    getInitialState: function() {
+      return {
+        hasMetrics: false,
```

Comment:
Given this is already in place and is reasonably explicit, I think it's probably fine.